### ISSUE TITLE
Error rendering component (mu-autocomplete)

### ISSUE THREAD

**Issue by kp-bit** [ironmansoftware/powershell-universal#2668]
### Version

4.0.12

### Severity

High

### Steps to Reproduce

After looking up an AD User and populating a $Session variable which is fed into a New-UDAutocomplete, it fails with the following error:

![image](https://github.com/ironmansoftware/issues/assets/61055573/61aae13e-47be-4210-b9d3-5fda9e4fe86c)

### Expected behavior

```console
Autocomplete is filled out with user name
```

### Actual behavior

```console
Error rendering component (mu-autocomplete)
TypeError: props.options.find is not a function

    at UDAutocomplete (https://server/Components_autocomplete_jsx.40f861593dda8fe3383f.bundle.js:62:73)
    at highOrderComponent (https://server/index.e024a3512509cd131d0b.bundle.js:89015:55)
    at ErrorBoundary (https://server/index.e024a3512509cd131d0b.bundle.js:87244:164)
    at Suspense
    at highOrderComponent (https://server/index.e024a3512509cd131d0b.bundle.js:89015:55)
    at ErrorBoundary (https://server/index.e024a3512509cd131d0b.bundle.js:87244:164)
    at Suspense
    at div
    at highOrderComponent (https://server/index.e024a3512509cd131d0b.bundle.js:89015:55)
    at ErrorBoundary (https://server/index.e024a3512509cd131d0b.bundle.js:87244:164)
    at Suspense
    at div
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at DialogContent (https://server/index.e024a3512509cd131d0b.bundle.js:14888:82)
    at div
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at Paper (https://server/index.e024a3512509cd131d0b.bundle.js:25851:83)
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at div
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at Transition (https://server/index.e024a3512509cd131d0b.bundle.js:327435:30)
    at Fade (https://server/index.e024a3512509cd131d0b.bundle.js:16546:77)
    at FocusTrap (https://server/index.e024a3512509cd131d0b.bundle.js:36100:5)
    at div
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at Portal (https://server/index.e024a3512509cd131d0b.bundle.js:37487:5)
    at Modal (https://server/index.e024a3512509cd131d0b.bundle.js:36437:7)
    at Modal (https://server/index.e024a3512509cd131d0b.bundle.js:24680:82)
    at https://server/index.e024a3512509cd131d0b.bundle.js:825:66
    at Dialog (https://server/index.e024a3512509cd131d0b.bundle.js:15274:83)
    at Modal (https://server/index.e024a3512509cd131d0b.bundle.js:81628:66)
    at div
    at Suspense
    at highOrderComponent (https://server/index.e024a3512509cd131d0b.bundle.js:89015:55)
    at Suspense
    at ThemeProvider (https://server/index.e024a3512509cd131d0b.bundle.js:41069:5)
    at ThemeProvider (https://server/index.e024a3512509cd131d0b.bundle.js:42449:5)
    at ThemeProvider (https://server/index.e024a3512509cd131d0b.bundle.js:39426:14)
    at StyledEngineProvider (https://server/index.e024a3512509cd131d0b.bundle.js:41244:5)
    at n (https://server/index.e024a3512509cd131d0b.bundle.js:339196:58)
    at https://server/index.e024a3512509cd131d0b.bundle.js:339196:1060
    at Dashboard (https://server/index.e024a3512509cd131d0b.bundle.js:88089:22)
    at Route (https://server/index.e024a3512509cd131d0b.bundle.js:324248:29)
    at Switch (https://server/index.e024a3512509cd131d0b.bundle.js:324454:29)
    at div
    at Router (https://server/index.e024a3512509cd131d0b.bundle.js:323879:30)
    at BrowserRouter (https://server/index.e024a3512509cd131d0b.bundle.js:323504:35)
    at QueryClientProvider (https://server/index.e024a3512509cd131d0b.bundle.js:322224:21)
    at n (https://server/index.e024a3512509cd131d0b.bundle.js:339196:58)
    at https://server/index.e024a3512509cd131d0b.bundle.js:339196:1060
    at App (https://server/index.e024a3512509cd131d0b.bundle.js:83730:89)
```

### Additional Environment data

Same behavior on both Dev and Prod servers.

**Comment by adamdriscoll** [comment]
@mattselle - Stole this one as I was already looking into this from a support case.

**Comment by adamdriscoll** [comment]
Resolved in 4.1.8
